helmfile/values/kube-prometheus-stack-wc.yaml.gotmpl

```diff
@@ -90,11 +88,11 @@ kube-state-metrics:
     - kubeadmcontrolplanes
   volumeMounts:
     - mountPath: /etc/config
-      name: capi-kube-state-metrics-volume
+      name: kube-state-metrics-clusterapi-volume
   volumes:
     - configMap:
-        name: capi-kube-state-metrics
-      name: capi-kube-state-metrics-volume
+        name: kube-state-metrics-clusterapi
+      name: kube-state-metrics-clusterapi-volume
```

Comment:
Should this not only be enabled if `.Values.kubeStateMetrics.clusterAPIMetrics.enabled` is true?

Reply:
That's how it works, see line 31

Reply:
Sorry, I was looking at the commit view so I didn't see that